src/material-experimental/mdc-chips/harness/chip-harness.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness} from '@angular/cdk-experimental/testing';
+
+/**
+ * Harness for interacting with a mat-chip in tests.
+ * @dynamic
+ */
+export class MatChipHarness extends ComponentHarness {
+  static hostSelector = 'mat-basic-chip, mat-chip';
+
+  /** Gets a promise for the text content the option. */
+  async getTextContent(): Promise<string> {
+    return (await this.host()).text();
+  }
+
+  /** Gets a promise for the disabled state. */
+  async isDisabled(): Promise<boolean> {
```

Comment:
I'm not sure about this one. A chip by itself doesn't have any interactive role, and thus being disabled doesn't make sense.

Reply:
Maybe we should move this to the `MatChipOptionHarness`? Should we not support `disabled` on the base `MatChip`?

Reply:
I moved it to the option, we can evaluate whether the MdcMatChip should have `disabled` as an input